.github/workflows/dist.yml

```
@@ -0,0 +1,76 @@
+name: DIST
+on:
+  push:
+    branches:
+      - main
+    tags:
+      - '*'
+  pull_request:
+    branches:
+      - '*'
```

I don't think we want to publish everything, just releases, right?
```suggestion
on:
  release:
    types: [ released, prereleased ]
```
With ``prereleased`` being optional. See https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#release